braindecode/models/eeginception_erp.py

```diff
@@ -14,10 +14,9 @@ def _transpose_to_b_1_c_0(x):
     return x.permute(0, 3, 1, 2)
 
 
-class EEGInception(nn.Sequential):
```

Comment:
you are here breaking backward compatibility as this model was shipped in a previous release. Can I ask you to properly deprecate it? thanks

Reply:
Sure!